src/main/java/ay2122s1_cs2103t_w16_2/btbb/logic/parser/order/AddOrderCommandParser.java

```diff
@@ -15,17 +18,31 @@
 public class AddOrderCommandParser implements Parser<AddOrderCommand> {
     @Override
     public AddOrderCommand parse(String args) throws ParseException {
-        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_CLIENT_PHONE);
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_CLIENT, PREFIX_CLIENT_PHONE,
```

Comment:
Maybe you change this to client index just for clarity & consistency
```suggestion
        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_CLIENT_INDEX, PREFIX_CLIENT_PHONE,
```